res/translations/tr/messages.ftl

```diff
@@ -4,7 +4,7 @@ welcome_message =
 
     Devam etmeden önce 3 şeye ihtiyacınız var:
     ✔ Ödülleri alacağınız bir cüzdan adresi (use Subwallet, polkadot{"{"}.js{"}"} uzantısı veya Substrate ağı ile uyumlu herhangi bir cüzdan kullanabilirsiniz)
-    ✔  Node verilerini depolamak için kaliteli bir SSD'de 100GB alan
+    ✔  Node verilerini depolamak için kaliteli bir SSD'de 100GiB alan
```

Comment:
And let's not use GiB here, use G like English translation does for consistency:
```suggestion
    ✔ Node verilerini depolamak için kaliteli bir SSD'de 100GiB alan
```